rites/roman1969/src/locales/fr.ts

```diff
@@ -152,19 +152,24 @@ export const locale: Locale = {
   },
 
   names: {
+    achard_of_saint_victor_bishop: 'Bienheureux Achard, évêque d’Avranches († 1171)', // src: mr_fr_1982_ed2_coutances
     adalbert_of_prague_bishop: 'Saint Adalbert, évêque de Prague et martyr († 997)',
     adalbert_of_prague_bishop_patron_of_poland:
       'Saint Adalbert, évêque de Prague, martyr et patron de la Pologne († 997)',
     adelaide_of_burgundy_empress: 'Sainte Adélaîde, impératrice († 999)',
+    adeline_de_mortain_abbess_and_the_saints_of_savigny:
+      'Sainte Adeline de Mortain († v. 1125) et les saints de Savigny', // src: mr_fr_1982_ed2_coutances
     adelphus_of_metz_bishop: 'Saint Adelphe, évêque († Vème s.)',
     agatha_of_sicily_virgin: 'Sainte Agathe, vierge et martyre en Sicile († 251)',
     agnes_of_rome_virgin: 'Sainte Agnès, vierge et martyre († v. 304)',
     albert_the_great_bishop:
       'Saint Albert le Grand, frère prêcheur, évêque de Ratisbonne, docteur de l’Église († 1280)',
     all_holy_bishops_of_the_archdiocese_of_lyon: 'Tous les Saints Évêques du diocèse de Lyon',
+    all_holy_bishops_of_the_diocese_of_coutances: 'Tous les Saints Évêques du diocèse de Coutances', // src: mr_fr_1982_ed2_coutances
```

Comment:
Has the celebration changed from `Les Saints Évêques de Coutances et Avranches` used in the source you shared with us? :thinking:

Reply:
![Screenshot 2024-10-11 at 11 25 31](https://github.com/user-attachments/assets/050dc3c0-a4a9-4d64-919b-dce419676097)

Reply:
I think we should omit `Tous` in order to use the official celebration name and add `et d’Avranches et les prêtres, leurs coopérateurs`. On the other hand, I am not sure if `du diocèse` should be removed, as IMHO it makes more sense to keep them, however, then we will not use the official celebration name. Maybe the celebration name writers chose to think about the episcopal sees/cities instead of dioceses. :wink:

Moreover, in titles, do you usually capitalise `Saints` and `Évêques` (under any grammatical rules that apply here)? Consider lowercasing these two words.

```suggestion
    all_holy_bishops_of_the_diocese_of_coutances: 'Les saints évêques du diocèse de Coutances et d’Avranches et les prêtres, leurs coopérateurs', // src: mr_fr_1982_ed2_coutances
```